README.md

```diff
@@ -6,6 +6,8 @@ Set `executionPlatform` and `executionPlatformVersion` in your BPMN diagrams.
 
 ## Usage
 
+### bpmn-js
```

Comment:
Let's also update the main description in the `README.md`

> Set executionPlatform and executionPlatformVersion in your BPMN and DMN diagrams.

Reply:
True.

Reply:
Done via https://github.com/bpmn-io/execution-platform/pull/3/commits/38a024d6f28591cdd0b82373bdbeb4b3ad88a0df